bundles/core/src/main/java/com/adobe/cq/forms/core/components/internal/servlets/FormMetaDataDataSourceServlet.java

```diff
@@ -29,11 +31,14 @@
 import javax.servlet.Servlet;
 import javax.servlet.ServletException;
 
+import com.day.crx.JcrConstants;
```

Comment:
JcrConstants is incorrectly imported

Reply:
import org.apache.jackrabbit.JcrConstants;